src/js/components/agencyV2/AgencyPage.jsx

```diff
@@ -102,8 +103,8 @@ export const AgencyProfileV2 = ({
         body: `View the spending activity for this state on USAspending.gov: ${getBaseUrl(slug)}`
     };
 
-    const handleShare = (str) => {
-        handleShareOptionClick(str, slug, emailArgs);
+    const handleShare = (optionName) => {
```

Comment:
(Not really related to this line) it looks like we need to add some padding between "Share" and "Download"
<img width="156" alt="Screen Shot 2021-04-12 at 10 13 51 AM" src="https://user-images.githubusercontent.com/7108785/114409012-00fea000-9b78-11eb-8cd8-d953e2a22ea7.png">